cortex/experiment/local.py

```diff
@@ -111,87 +119,123 @@ def reset(self):
 
     def last_run(self) -> Run:
+        """Returns the most recent run of this experiment
+
+        :return: _description_
+        :rtype: Run
+        """
         runs = self.runs()
         if len(runs) > 0:
             return runs[-1]
         return None
 
-    def find_runs(self, filter, sort, limit: int) -> List[Run]:
-        raise NotImplementedError('find_runs is not supported in local mode')
+    def find_runs(self, filter_obj, sort, limit: int) -> List[Run]:
+        """find_runs is not supported in local mode
+
+        :raises NotImplementedError: _description_
+        :rtype: List[Run]
+        """
+        raise NotImplementedError("find_runs is not supported in local mode")
 
     def _save_config(self):
         self._config.save(self._work_dir / self.config_file)
 
-    def load_artifact(self, run: Run, name: str, extension: str = 'pk'):
-        """
-        Returns a particular artifact created by the given run of the experiment.
+    def load_artifact(self, run: Run, name: str, extension: str = "pk") -> any:
+        """Returns a particular artifact created by the given run of the experiment.
 
         :param run: The run that generated the artifact requested.
+        :type run: Run
         :param name: The name of the artifact.
-        :param extension: An optional extension (defaults to 'pk').
+        :type name: str
+        :param extension: An optional extension, defaults to "pk"
+        :type extension: str, optional
+        :return: Python objects loaded into memory from the artifact content
+        :rtype: Any
         """
         artifact_file = self.get_artifact_path(run, name, extension)
-        with closing(open(artifact_file, 'rb')) as f:
-            return dill.load(f)
+        with closing(open(artifact_file, "rb")) as file_d:
+            return dill.load(file_d)
 
-    def get_artifact_path(self, run: Run, name: str, extension: str = 'pk'):
-        """
-        Returns the fully qualified path to a particular artifact.
+    def get_artifact_path(self, run: Run, name: str, extension: str = "pk") -> str:
+        """Returns the fully qualified path to a particular artifact.
 
         :param run: The run that generated the artifact requested.
-        :param name: The name of the artifact.
-        :param extension: An optional extension (defaults to 'pk').
-        """
-        return self._work_dir / self.dir_artifacts / "{}_{}.{}".format(name, run.id, extension)
+        :type run: Run
+        :param name:  The name of the artifact.
+        :type name: str
+        :param extension: An optional extension, defaults to "pk"
+        :type extension: str, optional
+        :return: A local filesystem path to the artifact
+        :rtype: str
+        """
+        return (
+            self._work_dir
+            / self.dir_artifacts
+            / "{}_{}.{}".format(name, run.id, extension)
+        )
 
     def _repr_html_(self):
         return _to_html(self)
 
     def display(self):
```

Comment:
Does this work ??

Reply:
Yeah. Did a quick test of the LocalExperiment module including logging params, metrics, artifacts, and works just fine:
```
>>> local.get_artifact_path(local.runs()[2], "blah")
PosixPath('/Users/pkandarpa/.cortex/local/experiments/test-exp/artifacts/blah_l4boeg1.pk')
```

Reply:
I couldn't find a file at this path however. Looking into this